rskj-core/src/main/java/co/rsk/validators/BlockTimeStampValidationRule.java

```diff
@@ -40,18 +40,28 @@ public BlockTimeStampValidationRule(int validPeriodLength) {
 
     @Override
     public boolean isValid(Block block) {
+        if (this.validPeriodLength == 0) {
+            return true;
+        }
+
         final long currentTime = System.currentTimeMillis() / 1000;
         final long blockTime = block.getTimestamp();
 
         boolean result = blockTime - currentTime <= this.validPeriodLength;
+
         if(!result) {
             logger.warn("Error validating block. Invalid timestamp {}.", blockTime);
         }
+
         return result;
     }
 
     @Override
     public boolean isValid(Block block, Block parent) {
+        if (this.validPeriodLength == 0) {
+            return true;
+        }
+
```

Comment:
Do OpenZeppelin test require to skip this validation ? I understand the one about blocks in the future, but not sure about this one.

Reply:
Yes, the validation interferes with truffle operations that advance the time for testing. Using 0 as period in regtest config, and the above added logic, allows truffle works in such scenarios

Reply:
Seems safe, but maybe we want to be extra careful: we could check if this is a `RegTestConfiguration`.

Reply:
@ajlopezrsk Thanks. Looks like there is a failing test.